### ISSUE TITLE
Block height cannot increase

### ISSUE THREAD

**Issue by oguzhane** [CityOfZion/neo-privatenet-docker#29]
I am trying to setup private neo blockchain using this repository but `create_wallet` step doesn't work. Until that phase everything went as expected but it prints `Waiting for wallet to sync..` message to screen **for hours**. I have no idea how to solve this problem. Unfortunately, there is no meaningful message for the issue.  

Any help or suggestion would be appreciated. Thank you!

**Comment by metachris** [comment]
It should take at most a few hours. Do you need to build it yourself? There is a ready to use image here: https://hub.docker.com/r/metachris/neo-privnet-with-gas/

If you really want to -- what operating system and version are you using? The CI system seems to be building it without problems using Ubuntu (and for me it works on Mac): https://travis-ci.org/CityOfZion/neo-privatenet-docker

**Comment by oguzhane** [comment]
My main system is Windows 10. I am running `Alpine Linux v3.7` on virtualbox. Docker is running on Alpine.
I have tried `neo-privnet-with-gas`. It is pulled and run without problem. I am accessing to the container for checking the wallet status. 
Used neo-python that is in `/opt/neo-python` directory in the container to open the wallet. Even though rebuilding wallet, wallet output shows synced_balances empty.
`wallet verbose` command output is as follows:

```
Wallet {
    "transactions": [],
    "percent_synced": 107,
    "claims": {
        "unavailable": 0.0,
        "available": 0.0
    },
    "addresses": [
        {
            "script_hash": "AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y",
            "tokens": null
        }
    ],
    "public_keys": [
        {
            "Address": "AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y",
            "Public Key": "031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a"
        }
    ],
    "path": "neo-privnet.wallet",
    "tokens": [],
    "height": 14,
    "synced_balances": [],
    "coins": []
}
```

**Comment by metachris** [comment]
Yeah, well if you create the image from scratch, this wallet is empty. It only works with the image from Docker hub, better use that. For what you're trying to do, there's not a reason to build it from scratch.
